server/server.go

```diff
@@ -11,14 +16,40 @@ type ContentResponseMeta struct {
 }
 
 type Content struct {
-	Body string `json:"body"`
+	ContentBody string `json:"content_body"`
 }
 
 type ContentResponse struct {
 	Meta ContentResponseMeta `json:"meta"`
 	Data []Content `json:"data"`
 }
 
+func requestFacebook(c echo.Context) error {
```

Comment:
Unless the function signature provides a parameter for describing the type of request and the type of endpoint, i think this function name needs to be much more specific to the type of resource being requested.